software/retrom.yml

```
@@ -0,0 +1,11 @@
+name: "Retrom"
+website_url: "https://github.com/JMBeresford/retrom"
+source_code_url: "https://github.com/JMBeresford/retrom"
+description: "Private cloud game library distribution server + frontend/launcher."
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+  - Rust
+tags:
+  - Games
```

```suggestion
  - Games - Administrative Utilities & Control Panels
```

More appropriate category

---

I found this category appropriate because [Romm](https://github.com/zurdi15/romm) is also in there and they seem quite similar. Should we move this to the other category as well?

---

Absolutely! I will do it now